src/test/Makefile

```
@@ -481,8 +481,8 @@ GTEST_DIR = ../../lib/test/gtest
 
 # Use clang/clang++ by default
 
-CC  := clang-12
-CXX := clang++-12
+CC  := clang-15
+CXX := clang++-15
```

Actually, it is a bit more complicated, `make` does define CC and CXX by default.

```suggestion
ifeq ($(origin CC), default)
CC  = clang-15
endif
ifeq ($(origin CXX), default)
CXX = clang++-15
endif
```